easybuild/easyconfigs/a/Affymetrix_Power_tools/apt-1.20.5.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
```

Comment:
@jdemeul Travis is complaining about the disparity between the subdirectory and the filename. If the subdirectory is `Affymetrix_Power_tools`, then the base filename should be the same. Conversely, the subdirectory should be `apt` if that's the name of the application.